### PR TITLE
artekki

### PR DESCRIPTION
**Pull request by artekki**
yooooooooooooooo

### CODE REVIEW

**Comment by feelsong** [comment]
sup

File: app/scripts/main.js [hunk 1]
```diff
@@ -156,7 +156,7 @@
 
 			tendrils.push(new Tendril({
 				//spring: 0.45 + 0.025 * (i / settings.trails)
-				spring: 0.6 + 0.001 * (i / settings.trails)
+				spring: 0.45 + 0.025 * (i / settings.trails)
```
Comment (feelsong) [inline app/scripts/main.js:159]: I can comment on this line

File: app/scripts/main.js [hunk 2]
```diff
@@ -167,8 +167,8 @@
 		ctx.fillStyle = 'rgba(8,5,16,0.5)';
 		ctx.fillRect(0, 0, ctx.canvas.width, ctx.canvas.height);
 		ctx.globalCompositeOperation = 'lighter';
-		//ctx.strokeStyle = 'hsla(' + Math.round(hue.update()) + ',90%,50%,0.1)';
-		ctx.strokeStyle = 'hsla(' + Math.round(hue.update()) + ',96%,50%,'+opacityUpdate()+')';
+		ctx.strokeStyle = 'hsla(' + Math.round(hue.update()) + ',90%,50%,0.1)';
+		//ctx.strokeStyle = 'hsla(' + Math.round(hue.update()) + ',50%,20%,'+opacityUpdate()+')';
```
Comment (feelsong) [inline app/scripts/main.js:171]: Why are you doing this????????

**Comment by artekki** [comment]
so pretty much , i work on this project and do pull request and comment it here. 
"thanks for the tutor" like this

File: app/scripts/main.js (hunk 1 above)
Comment (feelsong) [inline app/scripts/main.js:159]: this is funny

**Pull request merged**